Realm/ObjectServerTests/RLMObjectServerTests.mm

```diff
@@ -1277,7 +1280,7 @@ - (void)testDownloadAlreadyOpenRealm {
         NSUInteger sizeBefore = fileSize(c.pathOnDisk);
         XCTAssertGreaterThan(sizeBefore, 0U);
         XCTAssertNotNil(RLMGetAnyCachedRealmForPath(c.pathOnDisk.UTF8String));
-        [self waitForExpectationsWithTimeout:10.0 handler:nil];
+        [self waitForExpectationsWithTimeout:15.0 handler:nil];
```

Comment:
This particular timeout isn't problematic and I will remove it; like you said it was leftovers from experimentation.